contextualized/analysis/utils.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import numpy as np
+import pandas as pd
```

Comment:
I'm confused why this is the only addition to this file. Do we actually need pandas here?